users/pvinis/pvinis.h

```diff
@@ -0,0 +1,152 @@
+#pragma once
+
+#include "quantum.h"
+
+
+// my own keycodes
+enum userspace_custom_keycodes {
+  PV_ = SAFE_RANGE,
```

Comment:
This looks like you're using a placeholder.  You don't actually need to do that.  So this should be `PV_VRSN`, instead.

Reply:
I do use it as a placeholder. I think it's nice, one missed keycode should be ok to be used like that, right?